console/src/components/UserSelectionPopup/UserSelectionPopup.js

```diff
@@ -42,7 +42,14 @@ const UserSelectionPopupBase = kind({
 		}
 	},
 
-	render: ({userId, usersList, updateUser, resetUserSettings, resetAll, ...rest}) => (
+	handlers: {
+		onResetAll: (ev, {onReset, resetAll}) => {
+			onReset();
+			resetAll();
```

Comment:
I noticed that the active user's customizations aren't _visibly_ updated though they are cleared from localStorage. Perhaps we should also call `resetUserSettings` here as well. What do you think?